idaes/models/properties/general_helmholtz/components/parameters/__init__.py

```diff
@@ -0,0 +1,111 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory,
+# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
+# University, West Virginia University Research Corporation, et al.
+# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
+# for full copyright and license information.
+#################################################################################
+"""This module gets the path of parameters files
+"""
+
+__author__ = "John Eslick"
+
+import os
+import re
+import json
+
+from pyomo.common.fileutils import this_file_dir
+
+import idaes
+from idaes.models.properties.general_helmholtz.components import (
+    clear_component_registry,
+    register_helmholtz_component,
+)
+
+
+def get_parameter_path():
+    pth = idaes.cfg.properties.helmholtz.parameter_file_path
+    if pth is None:
+        pth = this_file_dir()
+    return pth
+
+
+def set_parameter_path(path):
+    idaes.properties.helmholtz.parameter_file_path = path
+
+
+def auto_register():
+    pth = get_parameter_path()
+    clear_component_registry()
```

Comment:
Just to check - this will not cause a problem with unregistering a user's files as they would immediately be re-registered?

Reply:
Should be okay.  I really don't expect this to be set on the fly.  Just once before you start using it, if you don't want the default location.